doc/tut2.rst

```diff
@@ -674,17 +674,18 @@ variable number of arguments:
 
   macro debug(n: varargs[expr]): stmt =
     # `n` is a Nim AST that contains a list of expressions;
-    # this macro returns a list of statements:
+    # this macro returns a list of statements (n is passed for proper line
+    # information):
     result = newNimNode(nnkStmtList, n)
     # iterate over any argument that is passed to this macro:
-    for i in 0..n.len-1:
+    for x in n:
```

Comment:
Just wanted a minimal patch to improve the 0..n.len-1 parameter. n looks fine to me. It's usage is also quite obvious.